public/components/overview/mitre_attack_intelligence/module_mitre_attack_intelligence_all_resources.tsx

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I think the name of the files are too long
`mitre_attack_intelligence` is repeated from the parent folder maybe is not necessary for the name of the file

Reply:
Reduced the length of file names